tests/ci/integration/librelp_patch/aws-lc-librelp.patch

```diff
@@ -0,0 +1,97 @@
+diff --git a/src/tcp.c b/src/tcp.c
+index 18cffda..110fcd0 100644
+--- a/src/tcp.c
++++ b/src/tcp.c
+@@ -276,7 +276,7 @@ int opensslh_THREAD_cleanup(void)
+  * alorbach, 2018-06-11
+  */
+ 
+-#if OPENSSL_VERSION_NUMBER >= 0x30000000L && !defined(LIBRESSL_VERSION_NUMBER)
++#if OPENSSL_VERSION_NUMBER >= 0x10100000L && !defined(LIBRESSL_VERSION_NUMBER)
+ long BIO_debug_callback_ex(BIO *bio, int cmd, const char LIBRELP_ATTR_UNUSED *argp,
+ 			   size_t LIBRELP_ATTR_UNUSED len, int argi, long LIBRELP_ATTR_UNUSED argl,
+ 			   int ret, size_t LIBRELP_ATTR_UNUSED *processed)
+@@ -1230,10 +1230,18 @@ relpTcpTLSSetPrio_ossl(relpTcp_t *const pThis)
+ {
+ 	char pristringBuf[4096];
+ 	char *pristring;
++	char errmsg[1024];
+ 	ENTER_RELPFUNC;
+ 	/* Compute priority string (in simple cases where the user does not care...) */
+ 	if(pThis->pristring == NULL) {
+ 		if (pThis->authmode == eRelpAuthMode_None) {
++			#if defined(OPENSSL_IS_AWSLC)
++			snprintf(errmsg, sizeof(errmsg),
++				"Warning: AWS-LC does not support anonymous ciphersuites.");
++			callOnErr(pThis, errmsg, RELP_RET_ERR_TLS);
```

Comment:
Can you just pass the string to callOnErr? There is no formatting going on in the call to snprintf.
```suggestion
+			callOnErr(pThis, "Warning: AWS-LC does not support anonymous ciphersuites.", RELP_RET_ERR_TLS);
```

Reply:
I'm guessing this would work as well, but I chose to follow along what the rest of `librelp` was doing. Here's some examples within the same file: https://github.com/rsyslog/librelp/blob/27d9a8cab814780d637d9857b8cd5aff39155a20/src/tcp.c#L1715-L1748